### PR TITLE
Merged cluster clean

### PR DESCRIPTION
**Pull request by rmohr**
Follow up for #708 to merge kubevirt cluster clean up into one place.

### CODE REVIEW

**Comment by rmohr** [comment]
@krsacme, FYI.

File: cluster/clean.sh [hunk 1]
```diff
@@ -31,11 +31,19 @@ _kubectl delete ds -l "kubevirt.io" -n kube-system --cascade=false --grace-perio
...
 
-# Delete everything, no matter if release, devel or infra
-_kubectl delete -f ${MANIFESTS_OUT_DIR}/ -R --grace-period 1 2>/dev/null || :
-
-# Delete exposures
-_kubectl delete services -l "kubevirt.io" -n kube-system
+# Delete all traces of kubevirt
+namespaces=(default kube-system)
+for i in ${namespaces[@]}; do
+    kubectl -n ${i} delete deployment -l 'kubevirt.io'
```
Comment (davidvossel) [inline cluster/clean.sh]: If we're wanting to ensure this cleans up all VMs and VM pods, we'd want to delete replica sets here as well.

Comment (rmohr) [reply cluster/clean.sh]: Added.

**Comment by rmohr** [comment]
retest this please

**Comment by davidvossel** [comment]
retest this please

**Comment by rmohr** [comment]
@davidvossel had an issue in the script.

**Comment by stu-gott** [comment]
retest this please

**Comment by davidvossel** [comment]
retest this please

**Comment by davidvossel** [comment]
@rmohr looks like the deployment is failing

**Comment by rmohr** [comment]
retest this please

**Comment by davidvossel** [comment]
I'd like this change to go in. The functional tests aren't able to run for some reason though.

**Comment by rmohr** [comment]
The cleanup patch seems to not clean everyting. Have to give it another iteration:

```
Error from server (AlreadyExists): error when creating "/var/lib/jenkins/workspace/kubevirt-functional-tests-vagrant-dev/go/src/kubevirt.io/kubevirt/_out/manifests/dev/rbac.authorization.k8s.yaml": clusterroles.rbac.authorization.k8s.io "kubevirt-controller" already exists
```

**Pull request merged**